swagger.yaml

```diff
@@ -8374,6 +8374,9 @@ definitions:
       is_success:
         type: boolean
         description: API VIP connectivity check result.
+      ignition:
```

Comment:
I don't see any change in the code, who set this value before the user get it from the API?

Reply:
It would be used by the agent: https://github.com/openshift/assisted-installer-agent/pull/263
Will push service change in an upcoming PR.

Reply:
You mean "agent change" right?!

Reply:
Meant the full validation logic:) https://github.com/openshift/assisted-service/pull/3074